test/test.lua

```diff
@@ -2567,10 +2578,13 @@ function test.logNormal()
    local tolerance = 0.01
    local t = torch.CudaTensor(sz1, sz2)
 
-   t:logNormal(mean, std)
-   local logt = t:log()
-   tester:assertalmosteq(logt:mean(), mean, tolerance, "mean is wrong")
-   tester:assertalmosteq(logt:std(), std, tolerance, "standard deviation is wrong")
+   for _, typename in ipairs(float_typenames) do
```

Comment:
test:
x:mean() == mean and x:std() == std
logx = torch.log(x)
logx:mean() == mean * mean / math.sqrt(std * std + mean * mean)
logx:std() == math.sqrt(math.log(std * std / mean * mean + 1)